build-parent/pom.xml

```diff
@@ -76,7 +76,7 @@
 
     <version.flyway>4.2.0</version.flyway>
 
-    <version.gradle>3.0</version.gradle>
+    <version.gradle>3.1</version.gradle>
```

Comment:
By bumping to 3.1, I am able to make use of an API (not available in 3.0) that is available all the way till 5+ version. The original API that I was making use of was marked as deprecated in 3.0 and finally removed in 5.0.